cranelift/codegen/src/loop_analysis.rs

```diff
@@ -35,9 +35,9 @@ struct LoopData {
 
 /// A level in a loop nest.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
-pub struct LoopLevel(u8);
+pub struct LoopLevel(u32);
```

Comment:
Ah, reverted, thanks for catching this!

For the curious, this change came from a dead-end (since reverted) where we invented a notion of a "pseudo-loop nest" based on the domtree (basic idea is that we wanted the loop nest to fully be a subtree of the domtree; consider e.g. a loop exit dominating the rest of the function, that's where the loop tree and domtree differ) and loop nests could get much deeper. But we've reverted all that and `LoopLevel` should correspond only to true loop-nest level again, now.